src/components/Community/CommunityPost.tsx

```
@@ -134,63 +228,54 @@ const CommunityPost = (/*{ boardType }: PostProps*/) => {
-                <label className="writer">·</label>
-                <label className="writer">회장</label>
+          {comments?.map((comment) => (
+            <>
+              <div className="reply">
+                <img src={comment.author.institutionImageUrl} />
+                <div className="labels">
+                  <div className="writer-wrapper">
+                    <label className="writer">
+                      {comment.author.authorName}
+                    </label>
+                    <label className="writer">·</label>
```

Biome (1.9.4)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)